lib/experimental/fonts-library/class-wp-font-family.php

```diff
@@ -0,0 +1,415 @@
+<?php
+/**
+ * Font Family class.
+ *
+ * This file contains the Font Family class definition.
+ *
+ * @package    Gutenberg
+ * @subpackage Fonts Library
+ * @since      X.X.X
+ */
+
+if ( class_exists( 'WP_Font_Family' ) ) {
+	return;
+}
+
+
+/**
+ * Fonts Library class.
+ */
+class WP_Font_Family {
+
+	/**
+	 * Font family data
+	 *
+	 * @var array
+	 */
+	private $data;
+
+
+	/**
+	 * WP_Font_Family constructor.
+	 *
+	 * @param array $font_family Font family data.
+	 */
+	public function __construct( $font_family = array() ) {
+		$this->data = $font_family;
+	}
+
+	/**
+	 * Returns the font family data.
+	 *
+	 * @return array An array in fontFamily theme.json format.
+	 */
+	public function get_data() {
+		return $this->data;
+	}
+
+	/**
+	 * Returns the font family data.
+	 *
+	 * @return string fontFamily in theme.json format as stringified JSON.
+	 */
+	public function get_data_as_json() {
+		return wp_json_encode( $this->get_data() );
+	}
+
+	/**
+	 * Returns if the font family has font faces defined.
+	 *
+	 * @return bool true if the font family has font faces defined, false otherwise.
+	 */
+	public function has_font_faces() {
+		return (
+			isset( $this->data['fontFace'] )
+			&& is_array( $this->data['fontFace'] )
+			&& ! empty( $this->data['fontFace'] )
+		);
```

Comment:
It looks like `$this->data['fontFace']` has to be a non-empty array for this to return `true`.

If that's the case, this should have the same result:
```php
return ! empty( $this->data['fontFace'] ) && is_array( $this->data['fontFace'] )
```

Reply:
done